src/content/getting-started/architecture/route-agent/_index.en.md

```diff
@@ -33,6 +33,50 @@ the new active Gateway Engine node.
 
 ## OVN Kubernetes
 
-For the OVN Kubernetes CNI plugin, host network routing is configured on all nodes and,
-on the active Gateway node, IP forwarding is configured between the `ovn-k8s-gw0`
-and cable interfaces.
+ With OVN Kubernetes we reuse the GENEVE tunnels created by OVNKubernetes CNI to reach the
+gateway nodes from non-gateway nodes and a separate VXLAN tunnel is not created.
+
+{{% notice info %}}
+For Submariner 0.15 and below refer  [network plugin syncer](../networkplugin-syncer/)
+{{% /notice %}}
+
+With OVN we can have two deployment models,
+
+{{% notice info %}}
+The Submariner automatically chooses the mode based on the deployment and this is not a
```

Comment:
Submariner automatically discovers the OVN mode and uses the appropriate implementation